StereoKit/Math/Bounds.cs

```diff
@@ -101,7 +101,41 @@ public bool Contains(Vec3 linePt1, Vec3 linePt2)
 		public bool Contains(Vec3 linePt1, Vec3 linePt2, float radius)
 			=> NativeAPI.bounds_capsule_contains(this, linePt1, linePt2, radius);
 
-		public static Bounds operator *(Bounds a, float b) { return new Bounds(a.center*b, a.dimensions*b); }
+		/// <summary>Scale this bounds position and extents by the given scale.
+		///  Modifies this bounds object.</summary>
+		/// <remarks>See <seealso cref="Scaled(float)"/> or <seealso cref="Scaled(Vec3)"/>
+		/// for a non-mutating alternative</remarks>
+		/// <param name="scale">Scale to apply</param>
+		public void Scale(float scale)
+		{
+			center *= scale;
+			dimensions *= scale;
+		}
+
+		/// <inheritdoc cref="Scale(float)"/>
+		public void Scale(Vec3 scale)
+		{
+			center *= scale;
+			dimensions *= scale;
+		}
+
+		/// <summary>Scale the bounds position and extents by the given scale.
+		/// Returns a new Bounds.</summary>
+		/// <remarks>This is equivalent to using the * operator: bounds * scale</remarks>
+		/// <param name="scale">Scale</param>
+		/// <returns>A new scaled bounds</returns>
+		[Pure]
```

Comment:
I'm not sure on my use of the [Pure] here as the attribute is never validated by the compiler, and not a guarantee. Feel free to remove it if its not fitting the style.

Reply:
Reading up on Pure, it sounds like they may be a little archaic, but [will produce warnings](https://www.infoq.com/news/2019/01/pure-attribute-net-core/) with some code analyzers if the return value of the Method is ignored! If so, that sounds like a great reason to use it :)